### PR TITLE
Limit json updates endpoint

### PR DESCRIPTION
**Pull request by mvr320**

### CODE REVIEW

File: webapp/src/Service/DOMJudgeService.php [hunk 1]
```diff
@@ -378,39 +378,48 @@ public function getUpdates(): array
...
-                ->andWhere('clar.sender is not null')
-                ->andWhere('clar.answered = 0')
-                ->setParameter('contest', $contest)
-                ->getQuery()->getResult();
-        }
+        $judgehosts = [];
+        $rejudgings = [];
+        $internal_errors = [];
+
+        if ($this->authorizationChecker->isGranted('ROLE_ADMIN') or $this->authorizationChecker->isGranted('ROLE_JURY')) {
```
Comment (nickygerritsen) [inline webapp/src/Service/DOMJudgeService.php:385]: admin also has the jury role so you only have to check that. Also you can use `$this->checkRole('jury')`

File: webapp/public/js/domjudge.js [hunk 2]
```diff
@@ -68,15 +68,17 @@ function sendNotification(title, options = {})
...
 
     var link = null;
     if ( typeof options.link !== 'undefined' ) {
         link = options.link;
         delete options.link;
     }
-    options['icon'] = domjudge_base_url + '/apple-touch-icon.png';
 
+    options['icon'] = domjudge_base_url + '/apple-touch-icon.png';
+
```
Comment (eldering) [inline webapp/public/js/domjudge.js:81]: Please be cautious introducing whitespace-only changes.

Comment (mvr320) [reply webapp/public/js/domjudge.js]: I think I fixed this

**Comment by mvr320** [comment]
> Added one comment. Also your commit from the previous PR is also here, probably we should only merge this after that one.

I think its fixed,

File: webapp/public/js/domjudge.js (hunk 2 above)
Comment (nickygerritsen) [reply webapp/public/js/domjudge.js]: The newline seems to have moved, the rest is now ok

**Pull request merged**